LCU.NET.Extras/App.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Legendary_Rune_Maker;
+using Ninject;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LCU.NET.Extras
+{
+    public static class LCUApp
```

Comment:
I don't really like this structure of a static class, Is there a better way?

Reply:
I need to access these properties from this project (LCU.NET.Extras), but I'm not adding a dependency with Rift.Services here.